README.md

```diff
@@ -220,6 +220,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [Pathway Technologies and Services](https://pathway.com.np/) | Thapathali, Kathmandu | Solutions and services in the field of software development, business/knowledge process outsourcing as well as consultancy and advisory services on information and communication technologies
 - [Peace Nepal DOT Com P. Ltd.](https://peacenepal.com.np/) | Kupondole, Lalitpur | Website and Software Development, SEO, Web Hosting, Web Graphic and Logo Design
 - [Perfect Active Solutions Pvt. Ltd.](https://pas.com.np/) | Old Baneshwor, Kathmandu | Website and Software Development for organizations operating in Nepal
+- [Periwin Solutions](https://periwin.com/) | Periwin Solutions is a Nepal-UK based company offering software development, mobile and web apps, outsourcing, digital marketing, and accounting consultancy
```

Comment:
@acharyasuyog Thanks for contributing. Can you also include the location of the company?